CommandLine/emake/OptionsParser.cpp

```diff
@@ -111,7 +111,7 @@ OptionsParser::OptionsParser() : _desc("Options")
     ("collision,c", opt::value<std::string>()->default_value("None"), "Collision System (Precise, BBox, None)")
     ("extensions,e", opt::value<std::string>()->default_value("None"), "Extensions (Alarms, Paths, Timelines, Particles, MotionPlanning, ttf, libpng, IniFilesystem, RegistrySpoof, Asynchronous, StudioPhysics, VirtualKeys, XRandR, XTEST, FileDropper, None)")
     ("compiler,x", opt::value<std::string>()->default_value(def_compiler), "Compiler.ey Descriptor")
-    ("enigma-root,s", opt::value<std::string>()->default_value(fs::current_path().string()), "Path to ENIGMA's sources")
+    ("enigma-root", opt::value<std::string>()->default_value(fs::current_path().string()), "Path to ENIGMA's sources")
```

Comment:
Are two character short aliases allowed? You could use `rt` as shortcut for this one.